ce/api/multimeta.py

```diff
@@ -159,5 +172,10 @@ def multimeta(sesh, ensemble_name="ce_files", model="", extras="", cell_methods=
             result.netcdf_variable_name
         ] = result.variable_long_name
         rv[unique_id]["units"][result.netcdf_variable_name] = result.units
+        print("multi_year_mean = {}".format(result.multi_year_mean))
```

Comment:
Is this a debug statement? Still wanted?

Reply:
Whoops, thanks for catching that!